### PR TITLE
[feature #2528] separate zh_hant to zh_hant_tw, zh_hant_hk and zh_han_sg

### LINKED ISSUE #2528: Separate App Strings zh_hant to Localised Traditional Chinese
### User Story

As a user from different regions using traditional Chinese (1. Hong Kong / Macau, 2. Taiwan, 3. Singapore / Malaysia), I want to see the localised indicating message.

### Description

*Type*: Feature

*Summary*: We need someone eloquent and versatile in different Traditional Chinese (including HK/Macau standard, Taiwan Standard, Singapore/Malaysia Standard) to separate the latest `zh_hant` strings in Status to localised string files, say `zh_hk`, `zh_tw`, `zh_sg`.

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
The three different traditional Chinese translation (`zh_hk`, `zh_tw`, `zh_sg`) ought to be as up-to-date as the English one.

#### Actual behavior
There is only `zh_hant` for all regions / countries using traditional Chinese.

### Solution
*Summary*: 
- [ ] Three new files: `zh_hk.cljs`, `zh_tw.cljs`, `zh_sg.cljs` need to be added to the directory [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations)
- [ ] How you translate is a matter of your preference. But noticed that, there are significant updates of the English file for the 0.9.12 release, with lots of strings both add and **REMOVED**. Make sure that the files are exact mirrors of each other, except in different languages.

### Additional Information
* Status version: 0.9.12+

### PR DESCRIPTION
**Pull request by foreseaz**
separate zh_hant to zh_hant_tw, zh_hant_hk and zh_hant_sg, update zh_hant

Fixes #2528 

### Summary:

[comment]: # Finish 5 main tasks for Chinese translations:
1. Update `zh_hant` to most updated `en` and `zh_hans`.
2. Separate `zh_hant` to `tw`, `hk`, `sg` based on [OpenCC standard](https://github.com/BYVoid/OpenCC).
3. Fix layout problems of [中英文之間需要增加空格, Space needed between English and Chinese Characters](https://github.com/sparanoid/chinese-copywriting-guidelines#%E4%B8%AD%E8%8B%B1%E6%96%87%E4%B9%8B%E9%96%93%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC) for `zh_hans`, `zh_hant`, `zh_hant_*`.
4. Make consistently using of `你(You)` instead of `您 (Sir/Madam)`
5. update `bots/browse/translations.js` `bots/console/translations.js` `bots/mailman/translations.js` `bots/transactor/translations.js` for the missing three Traditional Chinese objects

### Steps to test:
- Open directory `/src/status_im/translations/`
- Kindly find `zh_hans.cljs`, `zh_hant.cljs`, `zh_hant_hk.cljs`, `zh_hant_sg.cljs`, `zh_hant_tw.cljs` for details.

status: ready

### CODE REVIEW

**Review by foreseaz — COMMENTED** [review]
@andytudhope could you please review the PR. Thanks so much.

**Comment by andytudhope** [comment]
Will do @foreseaz, thank you. We just merged this PR: https://github.com/status-im/status-react/pull/2510, so you can pull latest `develop`, follow the instructions to add your 3 new locales and test if all the correct keys (i.e. commands) are there. Then I can get @ninadou to look over the actual translations for us. If you have issues, let me know, and I can help run those tests for you.

**Comment by foreseaz** [comment]
@andytudhope , I just added the keys [here](https://github.com/status-im/status-react/pull/2552/files#diff-640fd584c0f806e87bf9fbded11fef3b) based on PR #2510 , and also updated `bots/browse/translations.js` `bots/console/translations.js` `bots/mailman/translations.js` `bots/transactor/translations.js` for the missing three Traditional Chinese objects. Could you run the tests for me? Thanks!

File: src/status_im/i18n.cljs [hunk 1]
```diff
@@ -115,9 +118,9 @@
                              :zh-hans-cn  zh-hans/translations
```
Comment (andytudhope) [inline src/status_im/i18n.cljs:115]: Please organise the Chinese section here like so:
```
                             :zh          zh-hans/translations
                             :zh-hans     zh-hans/translations
                             :zh-hans-cn  zh-hans/translations
                             :zh-hans-mo  zh-hans/translations
                             :zh-hant     zh-hant/translations
                             :zh-hant-tw  zh-hant-tw/translations
                             :zh-hant-sg  zh-hant-sg/translations
                             :zh-hant-hk  zh-hant-hk/translations
                             :zh-hant-cn  zh-hant/translations
                             :zh-hant-mo  zh-hant/translations
                             :zh-wuu      zh-wuu/translations
                             :zh-yue      zh-yue/translations
```

**Comment by andytudhope** [comment]
If you then scroll down a bit in `i18n.cljs` to here, and add in your new supported languages in a sensisble order like so:
```
(def supported-locales (spec/assert ::locales #{:fr
                                                :zh
                                                :zh-hans
                                                :zh-hans-cn
                                                :zh-hans-mo
                                                :zh-hant
                                                :zh-hant-sg
                                                :zh-hant-hk
                                                :zh-hant-tw
                                                :zh-hant-mo
                                                :zh-hant-cn
                                                :sr-RS_#Cyrl
                                                :en
                                                :de
                                                :lt
                                                :sr-RS_#Latn
                                                :sr}))
```

And run `lein test-cljs` you will see that they all pass! 🎉 And we've added a bunch of new languages to our list! Amazing, amazing work 😄  So, next steps will be for @ninadou to review the content, for me to get a bounty assigned to this, and then we can merge!

**Comment by foreseaz** [comment]
@andytudhope Just re-organise the Chinese section locale and add new supported locales, thank you for being so supportive!

File: src/status_im/i18n.cljs (hunk 1 above)
Comment (ninadou) [reply src/status_im/i18n.cljs:115]: It said "we went looking everywhere, but couldn’t find those commits". So where  can I find the latest translations?

Comment (foreseaz) [reply src/status_im/i18n.cljs:115]: @ninadou I use `add --amend && push -f` to make the changes into one single commit, so the hash of commit at this message may not be available now, how about go [here](https://github.com/status-im/status-react/pull/2552/files)?

File: resources/js/bots/console/translations.js [hunk 2]
```diff
@@ -102,6 +102,87 @@ I18n.translations = {
...
+        phone_placeholder: '電話號碼',
+
+        confirm_description: '確認碼',
+        confirm_validation_title: '確認碼',
+        confirm_validation_description: '格式錯誤',
+
+        password_description: '密碼',
+        password_placeholder: '輸入密碼',
+        password_placeholder2: '請重新輸入密碼以確認',
+        password_error: '密碼應不少於6個字元。',
```
Comment (ninadou) [inline resources/js/bots/console/translations.js]: password_error: '密碼應不少於6個字符。',

File: src/status_im/i18n.cljs (hunk 1 above)
Comment (foreseaz) [reply src/status_im/i18n.cljs:115]: Hey @ninadou , sorry for the disturb, any updates that I should fix here?

**Review by andytudhope — APPROVED** [review]

**Comment by oskarth** [comment]
@andytudhope Can this be merged?

**Comment by andytudhope** [comment]
@oskarth I am just trying to get a sensible reply out of @ninadou about the actual translated content. As soon as she comes back to me with the OK we can merge - have tested and everything works well.

**Comment by ninadou** [comment]
I have pointed out some incorrect or confusing translations. But the author didn't reply me or change the codes. But most of them are correct, so I don't know what to do next...

**Comment by andytudhope** [comment]
@ninadou the only review you have done here is the comment `It said "we went looking everywhere, but couldn’t find those commits". So where can I find the latest translations?`

I see no translation change requests - please it's important that you do all reviews here in the open on github so that we can track it and don't get blocked by stuff like this. Where and what, specifically, did you ask to be changed so that @foreseaz can do it and we can merge.

**Review by ninadou — CHANGES_REQUESTED** [review]

File: resources/js/bots/console/translations.js [hunk 3]
```diff
@@ -78,16 +78,16 @@ I18n.translations = {
...
         confirm_description: '確認碼',
         confirm_validation_title: '確認碼',
         confirm_validation_description: '格式錯誤',
 
         password_description: '密碼',
-        password_placeholder: '鍵入您的密碼',
-        password_placeholder2: '重新鍵入您的密碼',
+        password_placeholder: '鍵入你的密碼',
+        password_placeholder2: '重新鍵入你的密碼',
         password_error: '密碼不得短於6個字元。',
```
Comment (ninadou) [inline resources/js/bots/console/translations.js]: password_error: '密碼不得短於6個字符。',

File: resources/js/bots/console/translations.js [hunk 4]
```diff
@@ -102,6 +102,87 @@ I18n.translations = {
...
+        phone_placeholder: '電話號碼',
+
+        confirm_description: '確認碼',
+        confirm_validation_title: '確認碼',
+        confirm_validation_description: '格式錯誤',
+
+        password_description: '密碼',
+        password_placeholder: '輸入密碼',
+        password_placeholder2: '請重新輸入密碼以確認',
+        password_error: '密碼應不少於6個字元。',
```
Comment (ninadou) [inline resources/js/bots/console/translations.js]: password_error: '密碼應不少於6個字符。',

File: resources/js/bots/console/translations.js [hunk 5]
```diff
@@ -159,7 +240,7 @@ I18n.translations = {
     },
     'zh-wuu': {
         phone_title: '发送电话号码',
```
Comment (ninadou) [inline resources/js/bots/console/translations.js]: phone_title: '發送電話號碼',

File: resources/js/bots/transactor/translations.js [hunk 6]
```diff
@@ -74,55 +74,94 @@ I18n.translations = {
...
-        send_title: '发送ETH',
+        send_title: '发送 ETH',
         send_description: '付款',
 
-        request_title: '请求ETH',
+        request_title: '请求 ETH',
         request_description: '要求付款',
         request_requesting: '正在请求 ',
 
         validation_title: '金額',
```
Comment (ninadou) [inline resources/js/bots/transactor/translations.js]: validation_title: '金额',

File: resources/js/bots/transactor/translations.js [hunk 7]
```diff
@@ -74,55 +74,94 @@ I18n.translations = {
...
-        validation_insufficient_amount: 'ETH余额不足 ('
+        validation_insufficient_amount: 'ETH 余额不足 ('
     },
     'zh-yue': {
-        send_title: '發送ETH',
+        send_title: '發送 ETH',
         send_description: '發送付款',
 
-        request_title: '徵求ETH',
+        request_title: '徵求 ETH',
```
Comment (ninadou) [inline resources/js/bots/transactor/translations.js]: request_title: '請求 ETH',

File: resources/js/bots/transactor/translations.js [hunk 8]
```diff
@@ -74,55 +74,94 @@ I18n.translations = {
...
         request_description: '徵求付款',
```
Comment (ninadou) [inline resources/js/bots/transactor/translations.js]: request_description: '請求付款',

File: resources/js/bots/transactor/translations.js [hunk 9]
```diff
@@ -74,55 +74,94 @@ I18n.translations = {
...
         request_requesting: '徵求中 ',
```
Comment (ninadou) [inline resources/js/bots/transactor/translations.js]: request_requesting: '請求中',

File: src/status_im/translations/zh_hans.cljs [hunk 10]
```diff
@@ -85,14 +85,14 @@
    :phone-number                          "电话号码"
    :update-status                         "更新你的状态..."
    :add-a-status                          "添加状态..."
-   :status-prompt                         "设置你的 Status。使用 #hastags 可以帮助其他人了解您，并且谈论您正在想些什么。"
+   :status-prompt                         "设置你的 Status。使用 #hastags 可以帮助其他人了解你，并且谈论你正在想些什么。"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs]: :status-prompt                         "设置你的 Status。使用 #hashtags 可以帮助其他人了解你，并且谈论你正在想些什么。"

**Review by ninadou — CHANGES_REQUESTED** [review]

**Comment by ninadou** [comment]
@andytudhope Can you see my comment now? I left the comments days ago but I didn't submit them. So I thought you can see them. Sorry for stuck here...

**Comment by andytudhope** [comment]
@foreseaz been pinging you relentlessly on Riot, but no response? Any chance that you could handle the reviews Nina has suggested so that we can merge this PR?

**Comment by foreseaz** [comment]
@andytudhope been offline on Riot these days, I will update asap.

**Comment by foreseaz** [comment]
@ninadou Updated according to reviews, thanks for the support.
@andytudhope Sorry for the late response on Riot (so many other IMs distracting 😞), I will try to keep tracking on Riot constantly.

Just updated and please review. Thanks, guys.

**Pull request merged**